spark-tsfile/src/main/scala/org/apache/iotdb/tsfile/package.scala

```diff
@@ -19,15 +19,23 @@
 
 package org.apache.iotdb
 
-import org.apache.spark.sql.{DataFrame, DataFrameReader, DataFrameWriter}
+import org.apache.spark.sql.{DataFrame, DataFrameReader, DataFrameWriter, SparkSession}
 
 package object tsfile {
-
   /**
     * add a method 'tsfile' to DataFrameReader to read tsfile
     */
   implicit class TsFileDataFrameReader(reader: DataFrameReader) {
-    def tsfile: String => DataFrame = reader.format("org.apache.iotdb.tsfile").load
```

Comment:
Is this interface removed: spark.read.tsfile("test.tsfile") ?

You can add an interface, but do not remove the old interface

Reply:
I use default parameters for upward compatibility, that is, you can still use spark.read.tsfile("test.tsfile")  to get an old form table